scripts/selecionar-tudo.js

```diff
@@ -7,12 +7,26 @@ function renderBotaoSelecionarTudo(customToolbar, document) {
 
 function criarBotaoSelecionarTudo() {
   const botaoSelecionarTudo = document.createElement("BUTTON");
-  botaoSelecionarTudo.classList.add('botao-selecionar-tudo');
+  botaoSelecionarTudo.id = "botao-selecionar-tudo";
+  botaoSelecionarTudo.className = "botao-selecionar-tudo";
   botaoSelecionarTudo.innerHTML = 'Selecionar Tudo';
   return botaoSelecionarTudo;
 }
 
+function toggleBotaoSelecionarTudo() {
+  const botaoSelecionarTudo = document.getElementById("botao-selecionar-tudo");
+  if(botaoSelecionarTudo.className == "botao-selecionar-tudo") {
+    botaoSelecionarTudo.className = "botao-desselecionar-tudo";
+    botaoSelecionarTudo.innerHTML = 'Desselecionar Tudo';
+  }
+  else {
+    botaoSelecionarTudo.className = "botao-selecionar-tudo";
+    botaoSelecionarTudo.innerHTML = 'Selecionar Tudo';
+  }
```

Comment:
```suggestion
  botaoSelecionarTudo.classList.toggle('botao-selecionar-tudo');
  botaoSelecionarTudo.classList.toggle('botao-desselecionar-tudo');
  botaoSelecionarTudo.innerHTML = 'Desselecionar Tudo';
  
  if (botaoSelecionarTudo.classList.contains('botao-selecionar-tudo')) {
    botaoSelecionarTudo.innerHTML = 'Selecionar Tudo';
  }
```
Trabalhar com "classList" é um pouco mais "safe", porque os botões podem possuir múltiplas classes (é o comum).
Nesse caso a gente pode usar a função "toggle", que faz basicamente o mesmo.